pykrige/core.py

```diff
@@ -54,6 +58,26 @@
 from scipy.optimize import minimize
 
 
+def great_circle_distance(lon1, lat1, lon2, lat2):
```

Comment:
This could use a proper doc string, indicating the type and the expected units of `lon1`, `lat1` and of the result. For instance something like,

    """" Calculate the great circle distance between two points

    Parameters
    ----------
    lat1 : float
          latitude of point A (degrees)
 
    [...] 
    Returns
    -------
    distance: float
         the great circle distance (degrees)
    """"